dasgo/requests.go

```diff
@@ -43,12 +43,12 @@ type EditGlobalApplicationCommand struct {
 	ApplicationID            Snowflake
 	CommandID                Snowflake
 	Name                     string                      `json:"name,omitempty"`
-	NameLocalizations        map[Flag]string             `json:"name_localizations,omitempty"`
+	NameLocalizations        map[Flag]string             `json:"name_localizations"`
```

Comment:
These maps are locale maps: https://discord.com/developers/docs/reference#locales. So change them to map[string]string. For example map["da"] = "insert danish word here".